src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -239,22 +239,34 @@ protected function denyAccessUnlessGranted($attributes, $subject = null, string
 
     /**
      * Returns a rendered view.
+     *
+     * @deprecated since Symfony 5.1, use renderTemplate() instead.
      */
     protected function renderView(string $view, array $parameters = []): string
+    {
+        trigger_deprecation('symfony/framework-bundle', '5.1', 'The "renderView()" method is deprecated, use "renderTemplate()" instead.', E_USER_DEPRECATED);
```

Comment:
3rd arg: `'The "%s" method is [...]`
4th arg: `__METHOD__`

Reply:
Thanks! Changes done.